run_data_measurements.py

```diff
@@ -268,11 +275,32 @@ def main():
         help="Whether to write out corresponding HTML files (Optional)",
     )
     parser.add_argument("--out_dir", default="cache_dir", help="Where to write out to.")
+    parser.add_argument(
+        "--overwrite_previous",
+        default=False,
+        required=False,
+        action="store_true",
+        help="Whether to overwrite a previous cache for these same arguments (Optional)",
+    )
 
     args = parser.parse_args()
     print("Proceeding with the following arguments:")
     print(args)
     # run_data_measurements.py -n hate_speech18 -c default -s train -f text -w npmi
+    dataset_cache_dir = f"{args.dataset}_{args.config}_{args.split}_{args.feature}"
+    try:
+        create_repo(dataset_cache_dir, organization="datameasurements", repo_type="dataset", private=True, token=HF_TOKEN)
```

Comment:
I would probably handle this differently, e.g., with the specific Exception type, and something like:

```
try:
    create_repo(dataset_cache_dir, organization="datameasurements", repo_type="dataset", private=True, token=HF_TOKEN)
except FileExistsError:
    if args.overwrite_previous:
        print("Overwriting previous cache.")
    else: 
        print("Already created a repo for this cache. Set --overwrite_previous if you wish to overwrite the repository. Returning.")
        return
except Exception as e:
    print("There is an error %s on the hub that is preventing repo creation." % e)
    print("Returning without computing the dataset cache.")
    return
```